test_workflows.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# if GALAXY_URL and GALAXY_USER_KEY are not set, try to set them from local untracked file
+[ ! $GALAXY_URL ] || [ ! $GALAXY_USER_KEY ] && source .secret.env
```

Comment:
these are set by jenkins and I think "you've exported these env vars" is a reasonable pre-req for running this script.

are you doing this in case someone wants to run it manually or so?

Reply:
yes, just for manual runs though if I run it manually I probably won't test all of them at once, so perhaps this belongs in `run_workflow_tests.sh`

Reply:
Sounds good!

Reply:
I took this out as well.  I'm not sure if it would be easier for anyone than exporting the variables they need prior to running the tests.